styles/go.dev/catppuccin.user.css

```diff
@@ -681,6 +681,257 @@
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    #editor-container {
+      #left-side {
+        .slide-content {
+          background-color: @base;
+
+          a {
+            color: @accent-color;
+          }
+
+          code {
+            background-color: @accent-color !important;
+            color: @crust;
+          }
+
+          h2, li, p {
+            color: @text;
+          }
+
+          pre {
+            background-color: @mantle;
+            color: @text;
+          }
+        }
+
+        .module-bar {
+          background-color: @mantle;
+          color: @text;
+
+          a {
+            color: @accent-color;
+          }
+        }
+      }
+
+      #right-side {
+        background-color: @base;
+
+        #explorer {
+          background-color: @mantle;
+        }
+
+        #file-editor {
+          .CodeMirror {
+            background-color: @base;
+
+            .CodeMirror-lines {
+              background-color: @base;
+              color: @text;
+
+              .CodeMirror-linenumber {
+                color: @overlay1;
+              }
+
+              span.cm-atom {
+                color: @blue;
+              }
+
+              span.cm-comment {
+                color: @peach;
+              }
+
+              span.cm-keyword {
+                color: @mauve;
+              }
+
+              span.cm-number {
+                color: @green;
+              }
+
+              span.cm-string {
+                color: @red;
+              }
+
+              span.cm-variable {
+                color: @text;
+              }
+            }
+          }
+
+          .CodeMirror-gutters {
+            background-color: @mantle;
+            border-right-color: @surface2;
+          }
```

Comment:
Thanks for theming all of this! I know syntax highlighting is a pain. We do theme CodeMirror quite a lot across userstyles (https://github.com/search?q=repo:catppuccin/userstyles%20codemirror&type=code) so I'll probably clean this up in a later PR by abstracting it away to just an import :)